src/view/prChangesTreeDataProvider.ts

```diff
@@ -47,6 +50,9 @@ export class PullRequestChangesTreeDataProvider extends vscode.Disposable implem
 
 		this._localFileChanges = fileChanges;
 		this._onDidChangeTreeData.fire();
+
+		// Workaround to expand the tree view
+		this._view.reveal(this.getChildren()[0]);
```

Comment:
Did you get this to work properly? In #750 I did the same thing but if `getChildren` is called by us and there is no real tree item bind to it, VSCode doesn't know how to expand to that node.

Reply:
oh no, I convinced myself this was working when it wasn't. I thought I had the view set to collapsed and then saw it expand on reload, but I must have always just had it open. Let me see if there's a way to store and reuse the result of getChildren for this

Reply:
actually this seems like the problem:

```ts
		if (typeof this.dataProvider.getParent !== 'function') {
			return Promise.reject(new Error(`Required registered TreeDataProvider to implement 'getParent' method to access 'reveal' method`));
		}
```

I'll try adding a `getParent`

Reply:
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View

Reply:
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View

Reply:
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View